osf/migrations/0121_add_waffle_flag_for_csrf_enforcement.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-07-23 16:49
+from __future__ import unicode_literals
+
+from django.db import migrations
+from waffle.models import Switch
+
+CSRF_SWITCH = 'enforce_csrf'
+
+def add_enforce_csrf_switch(state, schema):
```

Comment:
I don't think you actually need the migration; if the switch doesn't exist, `switch_is_active` will return `False`.